libtcmu.c

```diff
@@ -653,11 +1152,11 @@ void tcmulib_command_complete(
 
 		ent->rsp.scsi_status = SAM_STAT_CHECK_CONDITION;
 
-		buf[0] = 0x70;  /* fixed, current */
-		buf[2] = 0x5;   /* illegal request */
+		buf[0] = 0x70;	/* fixed, current */
+		buf[2] = 0x5;	/* illegal request */
 		buf[7] = 0xa;
 		buf[12] = 0x20; /* ASC: invalid command operation code */
-		buf[13] = 0x0;  /* ASCQ: (none) */
+		buf[13] = 0x0;	/* ASCQ: (none) */
```

Comment:
guess that's tabs vs space. I moved everything to kernel coding style last time you commented about it -- guess this is one of the leftovers.